src/main/java/com/fabriccommunity/thehallow/block/ColoredPumpkinBlock.java

```diff
@@ -0,0 +1,56 @@
+package com.fabriccommunity.thehallow.block;
+
+import com.fabriccommunity.thehallow.registry.HallowedBlocks;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.CarvedPumpkinBlock;
+import net.minecraft.block.PumpkinBlock;
+import net.minecraft.entity.ItemEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.sound.SoundCategory;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.world.World;
+
+public class ColoredPumpkinBlock extends PumpkinBlock {
+
+	private PumpkinColor color;
+
+	public ColoredPumpkinBlock(Settings settings, PumpkinColor color) {
+		super(settings);
+		this.color = color;
+	}
+
+	public boolean activate(BlockState blockState, World world, BlockPos blockPos, PlayerEntity playerEntity, Hand hand, BlockHitResult blockHitResult) {
+		ItemStack stack = playerEntity.getStackInHand(hand);
+
+		if (stack.getItem() == Items.SHEARS) {
```

Comment:
Do we have a shears item tag to replace this item identity check?

Reply:
The logic in the for normal pumpkin game acts similarly with the exact same identity check for Items.SHEARS.